sxw_soilwat.c

```diff
@@ -244,8 +243,11 @@ static void _update_productivity(void) {
     // Calculate the biomass contribution of a STEPWAT2 resource group relative
     // to its SOILWAT2 vegetation type
     ForEachGroup(g) {
-        if (GT(vegTypeBiomass[RGroup[g]->veg_prod_type - 1], 0.))
-            RGroup[g]->rgroupFractionOfVegTypeBiomass = bmassg[g] / vegTypeBiomass[RGroup[g]->veg_prod_type - 1];
+        // Get biomass of SOILWAT2 vegetation type for current resource group
+        tmp = vegTypeBiomass[RGroup[g]->veg_prod_type];
+
+        if (GT(tmp, 0.))
+            RGroup[g]->rgroupFractionOfVegTypeBiomass = bmassg[g] / tmp;
```

Comment:
@dschlaep - why did you add tmp here? Why not just use vegTypeBiomass[RGroup[g]->veg_prod_type]?

Reply:
@kpalmqui - No strong reason: I hoped that it would make it easier to read, particularly the division. I removed `tmp` with commit 97e3b35aad83545ea1587ccc7a8455ee9f3a482f